docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+title: CI/CD Integration
+description: Guide to how to run HTTP/Browser monitors from your CI/CD pipeline.
+
+In addition to the scheduled monitor runs, you can also trigger monitor runs using API. You can use this API to trigger monitors runs as part of your CI/CD pipeline and block the deployment if the runs fail. On failure of the run, you will be alerted on the configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:
+
+* Test the APIs, websites, and the user journeys in your PR and staging environments and annotate the pull requests with the results. 
+* Test the APIs, websites, and the user journeys in production immediately after deployment and alert when it fails.
+* Track and catch major changes in website metrics like page load time, page size, request count, 3rd Party API performance, etc introduced as part of code changes.
+
+## Run Monitor API
```

Comment:
Are there other uses of this API?  Would it make sense to have it have its own page in docs?

Reply:
I am not aware of any other use case other than invoking the monitor run during build or from CI/CD pipelines.